### PR TITLE
chore: Add Windows 10c and new k8s verion windows zip

### PR DESCRIPTION
**Pull request by mainred**

### CODE REVIEW

File: vhdbuilder/packer/windows-image.env [hunk 1]
```diff
@@ -4,6 +4,6 @@
 # CLI example to get the latest image version:
 #   az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-with-Containers-smalldisk:latest
 WINDOWS_2019_BASE_IMAGE_SKU=2019-Datacenter-Core-smalldisk
-WINDOWS_2019_BASE_IMAGE_VERSION=17763.1397.2008070242
+WINDOWS_2019_BASE_IMAGE_VERSION=17763.1518.2010132039
```
Comment (AbelHu) [inline vhdbuilder/packer/windows-image.env:7]: Does this contain KB4577667?
https://support.microsoft.com/en-us/help/4580390/windows-10-update-kb4580390
```
Before installing this update

Microsoft strongly recommends you install the latest servicing stack update (SSU) for your operating system before installing the latest cumulative update (LCU). SSUs improve the reliability of the update process to mitigate potential issues while installing the LCU. For general information about SSUs, see Servicing stack updates and Servicing Stack Updates (SSU): Frequently Asked Questions.

If you are using Windows Update, the latest SSU (KB4577667) will be offered to you automatically. To get the standalone package for the latest SSU, search for it in the Microsoft Update Catalog.
```

Comment (mainred) [reply vhdbuilder/packer/windows-image.env:7]: Not sure, but the latest SSU is mentioned in every xB, we did not install this specially.

**Review by AbelHu — APPROVED** [review]
Suggest to verify whether KB4577667 is installed in the latest base image.

File: vhdbuilder/packer/windows-image.env (hunk 1 above)
Comment (mainred) [reply vhdbuilder/packer/windows-image.env:7]: We do have KB4577667 installed.
```
Source        Description      HotFixID      InstalledBy          InstalledOn              
------        -----------      --------      -----------          -----------              
WIN-PQDIDH... Update           KB4578966                          10/13/2020 12:00:00 AM   
WIN-PQDIDH... Security Update  KB4577667                          10/13/2020 12:00:00 AM   
WIN-PQDIDH... Update           KB4580390     pkrvm3hk92izmmj\p... 10/21/2020 12:00:00 AM   
```

**Pull request merged**